browser/ui/views/sidebar/sidebar_container_view.cc

```diff
@@ -662,9 +662,11 @@ void SidebarContainerView::OnEntryShown(SidePanelEntry* entry) {
       auto side_bar_index = sidebar_model_->GetIndexOf(item);
       auto* controller = browser_->sidebar_controller();
       controller->ActivateItemAt(side_bar_index);
-      break;
+      return;
     }
   }
+
+  ShowSidebarAll();
```

Comment:
Ah, it's just for testing. don't need to these changes in this PR.

Reply:
Yeah, thanks. removed.